Makefile

```diff
@@ -211,6 +211,9 @@ runner-e2e-serial: bin/envsubst
 	hack/render-e2e-runner.sh
 	hack/test-e2e-runner.sh
 
+build-topics:
+	mkdir -p bin && go run tools/lstopics/lstopics.go > bin/topics.json
```

Comment:
can ship (COPY) the file in the tests image https://github.com/openshift-kni/numaresources-operator/blob/main/Dockerfile.tests or to the operator image https://github.com/openshift-kni/numaresources-operator/blob/main/Dockerfile?  otherwise I don't understand how to use this offline

Reply:
it is going to sit in the tests image: Dockerfile PR pending because depends on this one